packages/vc-handler-ld-local/src/types/types.ts

```diff
@@ -220,6 +225,11 @@ export interface IVerifyPresentationLDArgs {
    */
   presentationPurpose?: typeof ProofPurpose
 
+  /**
+   * Optional. The key handle ({@link IKey#kid}) from the internal database.
+   */
+  keyRef?: string
```

Comment:
Why is this needed to verify a presentation?

Reply:
Removed